gitlint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# It is only here for reference and backward compatibility, the source of truth
+# is Jenkinsfile
```

Comment:
This could be added into `Dockerfile` as the first target 🤔

Reply:
No, it is not possible, at least without changes, because we added `.git` folder to `.dockerignore`.